src/state/models/ui/preferences.ts

```diff
@@ -298,6 +254,14 @@ export class PreferencesModel {
     await this.rootStore.me.savedFeeds.updateCache(clearCache)
   }
 
+  async syncLegacyPreferences() {
+    if (this.legacyPreferences) {
```

Comment:
Do we need to also delete the legacy prefs from local storage so that `getLegacyPreferences` returns undefined, or is it enough to have removed them from `serialize`?

Reply:
It's enough to remove them from serialize